scripts/_deploy-UC.js

```diff
@@ -9,19 +9,22 @@ const path = require('path');
 const configRelativePath = process.env.CONFIG_PATH || 'config.json';
 const configPath = path.join(__dirname, '..' , configRelativePath);
 const config = require(configPath);
+const argsObject = require('../arguments.js');
 
 async function main() {
   const networkName = hre.network.name;
 
   // The config should have a deploy object with the network name as the key and contract type as the value
   const contractType = config["deploy"][`${networkName}`];
+  const args = argsObject[`${contractType}`]
```

Comment:
Good call about input validation. However, if the args variable is undefined because the contractType does not exist, the contract can still be deployed (albeit without custom constructor arguments). I will thus only log a warning to the developer rather than throw an error and also update the following line when declaring the constructorArgs variable: 
```javascript
  const constructorArgs = [ucHandler, ...(args ?? [])];
```
Which will resolve in constructorArgs taking the value of ucHandler when the args is undefined.